apps/namadillo/src/App/Staking/WithdrawalButton.tsx

```diff
@@ -14,18 +14,20 @@ import invariant from "invariant";
 import { useAtomValue, useSetAtom } from "jotai";
 import { TransactionPair, broadcastTx } from "lib/query";
 import { useCallback, useEffect } from "react";
-import { MyValidator } from "types";
+import { MyValidator, UnbondingValidator } from "types";
 
 type WithdrawalButtonProps = {
   myValidator: MyValidator;
+  unbondingStatus: UnbondingValidator;
```

Comment:
Can we have a name here more like `1:1`? In case something like `unbondingValidator: UnbondingValidator`